### PR TITLE
return err when rabin min less than 16

### PR DESCRIPTION
**Pull request by kjzz**
I want to [fix bug in ipfs](https://github.com/ipfs/go-ipfs/issues/5425#issue-357051412),so i need to update code in chunker.

### CODE REVIEW

**Comment by kjzz** [comment]
Hey @Stebalien , the pr is used for [ipfs issue](https://github.com/ipfs/go-ipfs/issues/5425#issue-357051412),can you help me review it?Thx a lot

File: parse.go [hunk 1]
```diff
@@ -52,7 +54,9 @@ func parseRabinString(r io.Reader, chunker string) (Splitter, error) {
 		if err != nil {
 			return nil, err
 		}
-
+		if min < 16 {
+			return nil,ErrRabinMin
```
Comment (magik6k) [inline parse.go]: needs `go fmt ./...`

File: parse_test.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	r := bytes.NewReader(randBuf(t, max))
+	chk1 := "rabin-18-25-32"
+	chk2 := "rabin-15-23-31"
+	_, err := parseRabinString(r, chk1)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+	_, err = parseRabinString(r, chk2)
+	if err == nil ||  err.Error() != ErrRabinMin.Error() {
+		t.Errorf("it should be a ErrRabinMin here.")
```
Comment (magik6k) [inline parse_test.go]: By convention error strings shouldn't end with a `.`

Comment (magik6k) [reply parse_test.go]: Also, you should be able to just do `err == ErrRabinMin`.

**Review by magik6k — COMMENTED** [review]
Thanks!

**Comment by kjzz** [comment]
Hello @magik6k , i have update the pr.please help me review it.Thx a lot

**Review by Stebalien — APPROVED** [review]

**Pull request merged**